KenticoOnboardingApplication/src/KenticoOnboardingApplication.Services/Services/GetItemService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Threading.Tasks;
+using KenticoOnboardingApplication.Contracts.Models;
+using KenticoOnboardingApplication.Contracts.Repositories;
+using KenticoOnboardingApplication.Contracts.Services;
+
+namespace KenticoOnboardingApplication.Services.Services
+{
+    internal class GetItemService : IGetItemService
+    {
+        private readonly IListRepository _repository;
+
+        public GetItemService(IListRepository respository) => _repository = respository;
```

Comment:
[See](https://imgflip.com/i/2nj3a8)
```suggestion
        public GetItemService(IListRepository repository) => _repository = repository;
```

Reply:
Do you prefer assignment statements being body-less? Consider tinkering the R# settings a bit so either constructors or even better the assignment statements are not suggested for stripping its body – it is definitely up to you and we can definitely discuss it in person…